cms/models/managers.py

```diff
@@ -68,10 +68,23 @@ def search(self, q, language=None, current_site_only=True):
         plugins = plugin_pool.get_all_plugins()
         for plugin in plugins:
             cmsplugin = plugin.model
-            if hasattr(cmsplugin, 'search_fields'):
+            if not (
+                hasattr(cmsplugin, 'search_fields') and
+                hasattr(cmsplugin, 'cmsplugin_ptr')
```

Comment:
yes, it's needed. A plugin can also be the bare base model (e.g a Template only plugin). In that case there is no ``cmsplugin_ptr``.